### DISCUSSION TITLE
How do we like our Wednesday Meetings?

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#140] [General]
How do we like the format of our Wednesday meetings? How can we better serve those who attend, and address what they are looking for? Is there work that needs to be done. 

Should we consider doing a separate backlog grooming session, maybe every two weeks? Would removing that from the Wednesday sessions afford us the ability to discuss other important action items?

**Comment by AdamFrisby** [comment]
It does have downsides (i.e. more work in preparation), but producing a 4-5 bullet summary or status update on each issue prior to the call (preferably 2-3 days) saves time on the call itself as you don't need to explain them as completely; that leaves more time on the call itself for feedback.
